src/Keyboard.h

```diff
@@ -110,13 +110,21 @@
 #define KEY_F23           0xFA
 #define KEY_F24           0xFB
 
+#define KEY_NUM_LOCK       0xDB
+#define KEY_SCROLL_LOCK    0xCF
```

Comment:
Both `KEY_NUM_LOCK` and `KEY_SCROLL_LOCK` are already defined earlier in the same file.

Reply:
Agreed, I'll remove. Sorry about that. Thanks for spotting.

Reply:
Correction applied to the source code, and tested.